go/vt/servenv/grpc_server.go

```diff
@@ -58,6 +59,9 @@ var (
 	// GRPCCA is the CA to use if TLS is enabled
 	GRPCCA = flag.String("grpc_ca", "", "ca to use, requires TLS, and enforces client cert check")
 
+	// GRPCAuthPluginImpl which auth plugin to use (at the moment now only grpc_static_auth is supported)
+	GRPCAuthPluginImpl = flag.String("grpc_auth_plugin_impl", "", "Which auth plugin implementation to use (grpc_static_auth)")
```

Comment:
I'd suggest shortening this to `-grpc_auth` and changing the value to `"static"`.